### ISSUE TITLE
[Server] Launch of new server and documentation

### ISSUE THREAD

**Issue by andersevenrud** [os-js/OS.js#527]
If you're not hanging around in the chat(s), you might have missed that a new backend is in development!

**NOTE: THIS WILL GET MERGED INTO `master` BY THE END OF NOVEMBER!!! Make sure to update your code**

You can read about it here: https://gitter.im/os-js/topics/topic/5818ce090b10738c73fe24e0/new-server-codebase-and-handler-abstraction

And look at the new Manual (draft) here: https://os.js.org/manual/

This is the checklist of all the stuff that has to be done before going into `master` (It is already in `development`):

- [x] **Feedback and bug-hunting**
- [x] Finish a couple of *TODO* and *FIXME* marks in the code
- [x] Test backward compability completely
- [x] Make sure 'admin' group is respected (I think I might forgot a check)
- [x] Take out all the debug stuff
- [x] Make sure *Package Management* still works
- [x] Make sure *User Management*  still works
- [x] Make sure Windows still works
- [x] Make sure Standalone still works
- [x] Convert applications to new API layout (for all repos)
- [ ] Redo NW integration (**postponed**)
- [x] Make sure new manual (documentation) makes sense
- [x] Create a new os.js.org project
- [x] Make new manuals, tuorials etc
- [x] Redo jsdoc generation **(now in os.js.org repo)**
- [x] Make 302 redirects for old manual pages etc **(prepared)**
- [x] Robots.txt for 302 pages
- [x] Update unit tests
- [x] Document templating

## Deploy Checklist

**CURRENTLY DEPLOYING**

- [x] Create new `gh-pages` orphan branch
- [x] Create new `os.js.org` repository
- [x] Make sure the new manual is default in all docs
- [x] Make sure documentation is correct
- [x] Transfer `os.js.org` build files into `gh-pages`
- [x] Make sure unit tests run
- [x] Go through standard testing routine
- [x] Rebase `development` into `master`
- [x] Push `gh-pages`
- [x] Push `master`
- [x] Make a new signed github release and version bump
- [x] Publish article on community site
- [x] Update gitter announcement

**Comment by andersevenrud** [comment]
This is now done guys!

Enjoy :)
